app/next-client-app/app/datasets/page.tsx

```diff
@@ -67,6 +67,7 @@ export default async function DataSets({ searchParams }: DataSetListProps) {
               data={dataset.results}
               count={dataset.count}
               filter="name"
+              linkPrefix=""
```

Comment:
You can take this out as well as the other empty string `linkPrefix`

Reply:
If I remove this, the URL on the browser will show "undefined" for this part. So I needed it there, I guess. Can you please check on your end as well?